lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -95,12 +95,12 @@ sub latest_jobs {
     my @latest;
     my %seen;
     foreach my $job (@jobs) {
-        my $distri  = $job->DISTRI;
-        my $version = $job->VERSION;
-        my $build   = $job->BUILD;
-        my $test    = $job->TEST;
-        my $flavor  = $job->FLAVOR || 'sweet';
-        my $arch    = $job->ARCH || 'noarch';
+        my $distri  = $job->DISTRI  || 'nodistri';
```

Comment:
Yes, it should. But you need to add a script to set 'nodistri' to those currently being NULL. There are some :(